app/grandchallenge/evaluation/models.py

```diff
@@ -1315,6 +1358,14 @@ def assign_permissions(self):
         assign_perm("view_evaluation", admins_group, self)
         assign_perm("change_evaluation", admins_group, self)
 
+        if self.submission.phase.external_evaluation:
+            external_evaluators = (
+                self.submission.phase.challenge.external_evaluators_group
+            )
+            assign_perm("view_evaluation", external_evaluators, self)
+            assign_perm("change_evaluation", external_evaluators, self)
```

Comment:
We probably should keep the "change_evaluation" permissions separate as this allows different existing functionality. We want to minimise the scope of what the external evaluator can do as these will be service accounts.